posts/2024-04-26-ArgoCD-Configuration-Drift.adoc

```diff
@@ -71,7 +71,16 @@ To get the default Admin password, run the following command:
 kubectl get secret argocd-initial-admin-secret -n argocd -o jsonpath='{.data.password}' | base64 -d
```

Comment:
does this command return the default admin password as console output that the user copies and enters on the UI login page?

Reply:
ok- I revised this section slightly, just need to make sure I've understood correctly. You can review the update on the draft site:

https://blogs-draft-openlibertyio.mqj6zf7jocq.us-south.codeengine.appdomain.cloud/blog/2024/04/26/ArgoCD-Configuration-Drift.html#setup_argocd